packages/core/test/oracle/Voting.js

```diff
@@ -1793,4 +1808,302 @@ contract("Voting", function(accounts) {
     // After retrieval, the length should be decreased back to 0 since the element added in this test is now deleted.
     assert.equal((await votingTest.getPendingPriceRequestsArray()).length, 0);
   });
+  it("Votes can correctly handel arbitrary ancillary data", async function() {
+    const identifier1 = web3.utils.utf8ToHex("request-retrieval1");
+    const time1 = "1000";
+    const ancillaryData1 = web3.utils.utf8ToHex("some-random-extra-data"); // ancillary data should be able to store any extra dat
+
+    const identifier2 = web3.utils.utf8ToHex("request-retrieval2");
+    const time2 = "2000";
+    const ancillaryData2 = web3.utils.utf8ToHex(`callerAddress:${account4}`); // ancillary data should be able to store addresses
```

Comment:
nit: I think it would make this test slightly better if we made the identifier and timestamp match between both requests and only have the data differ. Then, this test would verify that the DVM correctly differentiates.

Thoughts? I think it would only require us to set identifier2 and time2 to identifier1 and time1 on these lines.

Reply:
good call. Have updated the tests accordingly and added a comment to explain why.

Reply:
+1 -- thanks @chrismaree!